esrally/driver/runner.py

```diff
@@ -501,6 +504,18 @@ async def __call__(self, es, params):
             bulk_params["timeout"] = params["timeout"]
         if "pipeline" in params:
             bulk_params["pipeline"] = params["pipeline"]
+        if "refresh" in params:
+            refresh_params = {"sync": "wait_for", "async": "true", "default": "false"}
```

Comment:
I think we just use a 1:1 mapping of the values to the ES API, any 'dynamic' behaviour can use a parameter source to set those values.